packages/clerk-js/src/utils/__tests__/captcha.spec.ts

```
@@ -23,3 +23,229 @@ describe('shouldRetryTurnstileErrorCode', () => {
+        nonce: 'test-nonce-from-csp',
+        openModal: async () => {},
+        siteKey: 'test-site-key',
+        widgetType: 'invisible',
+      };
+
+      // If this compiles without TypeScript errors, the test passes
+      expect(validOptions.nonce).toBe('test-nonce-from-csp');
+    });
+
+    it('should allow undefined nonce in CaptchaOptions', () => {
+      const validOptionsWithoutNonce: import('../captcha/types').CaptchaOptions = {
```

ESLint

[error] `import()` type annotations are forbidden.

(@typescript-eslint/consistent-type-imports)